libs/zalando-postgres/config.jsonnet

```diff
@@ -0,0 +1,43 @@
+local name = 'zalando-postgres';
```

Comment:
I'm going to be arrogant here and say you can claim the 'postgress-operator' name.

I came across the https://github.com/CrunchyData/postgres-operator but that has started ~3 months later. First come, first serve? :innocent:

Reply:
I would prefer to maybe call it `zalando-postgres-operator`. What do you think?

Reply:
Yeah, sounds fine. I'm even thinking we should drop the `-operator` suffix as this library won't install the operator.